examples/containers/multi/stop.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Copyright 2021 Couchbase, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file  except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the  License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -u
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# Delete ALL containers with the cbs_server_exp image
+docker ps -a --filter "ancestor=cbs_server_exp" --format '{{.ID }}' | xargs docker rm -f
```

Comment:
probably want a `docker rm -fv`

Reply:
To improve logging? Initially redirected the output to `/dev/null`

Reply:
No to remove volumes

Reply:
This is handled by including `--rm` in the `docker run` command.

Reply:
Does that remove volumes though? (And more to the point, do the containers we spin up use volumes?)

Reply:
I double checked, https://docs.docker.com/engine/reference/run/#clean-up---rm
> If you set the --rm flag, Docker also removes the anonymous volumes associated with the container when the container is removed... Only volumes that are specified without a name are removed.

The Couchbase Server/exporter image shouldn't, and any that the CMOS stack does should be handled by the existing `docker-compose down ...`